packages/twenty-server/src/modules/messaging/blocklist-manager/jobs/messaging-blocklist-reimport-messages.job.ts

```diff
@@ -46,11 +44,12 @@ export class BlocklistReimportMessagesJob
       return;
     }
 
-    await this.gmailFullMessageListFetchService.fetchConnectedAccountThreads(
-      workspaceId,
-      connectedAccount[0].id,
-      [handle],
-    );
+    // TODO: reimplement that
```

Comment:
Are we ok to merge it without this? 🤔

Reply:
@bosiraphael it's completely reworking it so I did not want to lose time here